File: sdkwork-app-sdk-typescript/src/types/prize-claim-vo.ts
/** 奖品领取结果 */
export interface PrizeClaimVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 是否领取成功 */
  success?: boolean;
  /** 奖品名称 */
  prizeName?: string;
  /** 奖品类型 */
  prizeType?: string;
  /** 领取时间 */
  claimTime?: string;
  /** 到账内容 */
  rewardDetails?: Record<string, unknown>;
  /** 提示消息 */
  message?: string;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/prize_claim_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PrizeClaimVO:
    """奖品领取结果"""
    created_at: str = None
    updated_at: str = None
    success: bool = None
    prize_name: str = None
    prize_type: str = None
    claim_time: str = None
    reward_details: Dict[str, Any] = None
    message: str = None
